src/main/java/org/dinosaur/foodbowl/domain/review/presentation/ReviewControllerDocs.java

```diff
@@ -0,0 +1,39 @@
+package org.dinosaur.foodbowl.domain.review.presentation;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Size;
+import java.util.List;
+import org.dinosaur.foodbowl.domain.member.domain.Member;
+import org.dinosaur.foodbowl.domain.review.dto.request.ReviewCreateRequest;
+import org.dinosaur.foodbowl.global.exception.ExceptionResponse;
+import org.dinosaur.foodbowl.global.presentation.Auth;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestPart;
+import org.springframework.web.multipart.MultipartFile;
+
+@Tag(name = "리뷰", description = "리뷰 API")
+public interface ReviewControllerDocs {
+
+    @Operation(summary = "리뷰 등록", description = "가게에 해당하는 리뷰를 등록합니다.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "201", description = "리뷰 등록 성공"),
+            @ApiResponse(
+                    responseCode = "400",
+                    description = "사진이 4개보다 많은 경우, 가게 이름 값이 없는 경우, 가게 주소 필드가 값이 없는 경우, "
+                            + "가게 경도 값이 없는 경우, 가게 위도 값이 없는 경우, 가게 정보 URL이 없는 경우, "
+                            + "가게 카테고리 정보가 없는 경우, 가게 리뷰 내용이 없는 경우",
+                    content = @Content(schema = @Schema(implementation = ExceptionResponse.class))
+            )
+    })
+    ResponseEntity<Void> create(
+            @RequestPart(name = "request") @Valid ReviewCreateRequest reviewCreateRequest,
+            @RequestPart(name = "images", required = false)
+            @Size(max = 4, message = "사진의 개수는 최대 4개까지 가능합니다.") List<MultipartFile> imageFiles,
+            @Auth Member member);
+}
```

Comment:
```suggestion
    ResponseEntity<Void> create(ReviewCreateRequest reviewCreateRequest, List<MultipartFile> imageFiles, Member member);
}
```
문서화를 위한 인터페이스이기에 구현체에 대한 정보는 빼는 것은 어떨까요~~

Reply:
상속 관계의 구조에서는 Bean validation이 필요한 컨트롤러는 동일하게 설정해주어야 합니다!
BeanValidation 레퍼런스에 나와있더라구욥

설정을 제거하면 다음과 같은 예외가 발생합니다.
```java
ConstraintDeclarationException: HV000151: A method overriding another method must not redefine the parameter constraint configuration, but method ReviewController#create(ReviewCreateRequest, List, Member) redefines the configuration of ReviewControllerDocs#create(ReviewCreateRequest, List, Member)
```

Reply:
보투게더에서는 어떻게 하셨나요?!!? 궁금하네용

Reply:
말씀하신대로 `Bean Validation`은 인터페이스에도 동일하게 적용해주어야 에러가 발행하지 않더라고요. 덕분에 배워갑니다 ㅎㅎ

`Bean Validation`인 `@Size`를 제외하고는 제거해도 동작하지 않나요? 🤔 보투게더에서도 문서화를 위한 인터페이스이기에 이외의 정보는 최대한 제거하는 방향으로 진행했습니다!

Reply:
DTO를 검증하는 `@Valid`외 `@Size` 두개만 있으면 되네요 ㅎㅎ

Reply:
`@Valid`도 없어도 되지 않나요? (진짜 모름)

Reply:
`@Valid`는  Bean Validation이 아니어서 상관없을 것 같았는데, 제거하니까 예외가 발생하더라구요 🥲

`HV000151: A method overriding another method must not redefine the parameter constraint configuration, but method ReviewController#create(ReviewCreateRequest, List, Member) redefines the configuration of ReviewControllerDocs#create(ReviewCreateRequest, List, Member)`